extensions/scala/api/src/main/scala/org/virtuslab/intellij/scala/protocol/SbtEndpoints.scala

```diff
@@ -0,0 +1,14 @@
+package org.virtuslab.intellij.scala.protocol
+
+import org.virtuslab.ideprobe.jsonrpc.JsonRpc.Method.Request
+import org.virtuslab.ideprobe.protocol.ProjectRef
+import org.virtuslab.ideprobe.jsonrpc.PayloadJsonFormat._
+import pureconfig.generic.auto._
+
+object SbtEndpoints {
```

Comment:
I would replace naming in this PR from Sbt to Scala as it is scala-probe and sbt is just a part of it. We may want to add more endpoints that will be scala specific but not sbt specific.

Reply:
done